KicadModTree/nodes/specialized/PadArray.py

```diff
@@ -193,7 +199,25 @@ def _createPads(self, **kwargs):
             else:
                 kwargs['shape'] = padShape
 
-            pads.append(Pad(number=number, at=[x_pad, y_pad], **kwargs))
+            if i == 0 and kwargs.get('chamfer_corner_selection_first') is not None\
```

Comment:
'chamfer_corner_selection_first' in kwars

Reply:
We have a lot of places where we checked kwargs.get against None instead of checking 'key' in kwargs. I will go around and fix this at least in the places where i touched it in this PR.